src/Compilers/CSharp/Test/Emit3/Semantics/OutVarTests.cs

```diff
@@ -32032,21 +32068,21 @@ public static int TakeOutParam<T>(T y, out T x)
                 var compilation = CreateCompilationWithMscorlib461(source, options: TestOptions.ReleaseExe, parseOptions: TestOptions.Regular9);
 
                 compilation.VerifyDiagnostics(
-                    // (3,12): error CS0116: A namespace cannot directly contain members such as fields or methods
+                    // (3,12): error CS9347: A compilation unit cannot directly contain members such as fields, methods or properties 
                     // fixed bool a[2], b[H.TakeOutParam(1, out var x1)];
-                    Diagnostic(ErrorCode.ERR_NamespaceUnexpected, "a").WithLocation(3, 12),
+                    Diagnostic(ErrorCode.ERR_CompilationUnitUnexpected, "a").WithLocation(3, 12),
                     // (3,18): error CS1642: Fixed size buffer fields may only be members of structs
                     // fixed bool a[2], b[H.TakeOutParam(1, out var x1)];
                     Diagnostic(ErrorCode.ERR_FixedNotInStruct, "b").WithLocation(3, 18),
-                    // (3,20): error CS0133: The expression being assigned to '<invalid-global-code>.b' must be constant
-                    // fixed bool a[2], b[H.TakeOutParam(1, out var x1)];
-                    Diagnostic(ErrorCode.ERR_NotConstantExpression, "H.TakeOutParam(1, out var x1)").WithArguments("<invalid-global-code>.b").WithLocation(3, 20),
                     // (3,12): error CS1642: Fixed size buffer fields may only be members of structs
                     // fixed bool a[2], b[H.TakeOutParam(1, out var x1)];
                     Diagnostic(ErrorCode.ERR_FixedNotInStruct, "a").WithLocation(3, 12),
                     // (3,12): error CS0214: Pointers and fixed size buffers may only be used in an unsafe context
                     // fixed bool a[2], b[H.TakeOutParam(1, out var x1)];
                     Diagnostic(ErrorCode.ERR_UnsafeNeeded, "a[2]").WithLocation(3, 12),
+                    // (3,20): error CS0133: The expression being assigned to '<invalid-global-code>.b' must be constant
```

Comment:
>// (3,20): error CS0133: The expression being assigned to '<invalid-global-code>.b' must be constant [](http://example.com/codeflow?start=20&length=100)

Consider putting this error into its original position to avoid unnecessary diff.